modules/openid/cms_openid_module.e

```diff
@@ -203,7 +203,7 @@ feature -- Hooks
 				until
 					lnk2 /= Void
 				loop
-					if ic.item.location.same_string ("account/roc-logout") then
+					if ic.item.location.same_string ("account/roc-logout") or else ic.item.location.same_string ("basic_auth_logoff")  then
```

Comment:
after the addition of the new module, I note the issue but I will need to double check if there was present before
the addition.